### ISSUE TITLE
bug (universal.sh): root owned .ssh and .local directories

### ISSUE THREAD

**Issue by xavierchanth** [atsign-foundation/noports#1142]
I'm really not sure how this happened, running fedora 40 with the systemd install.

Setup sshnpd first, then realized I didn't have openssh-server installed so I went back and added it to the machine.

![image](https://github.com/atsign-foundation/noports/assets/33691921/87adbd42-fc18-4d2a-a9aa-db66b5f2ec76)

Both `.local/` and `.ssh/` were owned by root. We also had a similar issue with .sshnp earlier today.

I'm thinking we should add a step to universal.sh which executes `mkdir -p <dir>` on all directories we use (.atsign, .local/bin, .sshnp, .ssh) at the beginning, and then `chown -R user:user <dir> | chown -R user <dir>` towards the end of installation.

**Comment by cpswan** [comment]
I just did a device install using 5.3.0 universal.sh onto a freshly installed (default) Fedora 40 Server:

![image](https://github.com/atsign-foundation/noports/assets/478926/ed5cb92a-663f-4f36-a260-299513276b6a)

.ssh is certainly a problem. In my case .local is as expected.

**Comment by cpswan** [comment]
Looks like the problematic piece is in `install.sh` rather than `universal.sh`:

```sh
# SETUP AUTHORIZED KEYS #

setup_authorized_keys() {
  mkdir -p "$user_ssh_dir"
  touch "$user_ssh_dir/authorized_keys"
  chown $user:$user "$user_ssh_dir/authorized_keys"
  chmod 644 "$user_ssh_dir/authorized_keys"
}
```

**Comment by Xlin123** [comment]
was this closed in #1153 ? @XavierChanth

**Comment by xavierchanth** [comment]
> was this closed in #1153 ? @XavierChanth

Should be
